lib/scsi_defs.h

```diff
@@ -2,9 +2,29 @@
 #define LIB_SCSI_DEFS_H
 
 #include <cstdint>
+#include "absl/base/attributes.h"
 
 namespace scsi_defs {
 
+  constexpr uint32_t kRead6CommandSize = 6;
+  constexpr uint32_t kRead10CommandSize = 10;
+  constexpr uint32_t kRead12CommandSize = 12;
+  constexpr uint32_t kRead16CommandSize = 16;
+  constexpr uint32_t kRead32CommandSize = 32;
+  constexpr uint32_t kWrite6CommandSize = 6;
+  constexpr uint32_t kWrite10CommandSize = 10;
+  constexpr uint32_t kWrite12CommandSize = 12;
+  constexpr uint32_t kWrite16CommandSize = 16;
+  constexpr uint32_t kWrite32CommandSize = 32;
+  constexpr uint32_t kVerify6CommandSize = 6;
+  constexpr uint32_t kVerify10CommandSize = 10;
+  constexpr uint32_t kVerify12CommandSize = 12;
+  constexpr uint32_t kVerify16CommandSize = 16;
+  constexpr uint32_t kVerify32CommandSize = 32;
+  constexpr uint32_t kSynchronizeCache10Size = 10;
+  constexpr uint32_t kSynchronizeCache16Size = 16;
```

Comment:
I just realized that we can club all of these together since they are all standard byte sizes. Any suggestions on naming?

Reply:
I'm not sure what you mean by "club together". In this particular case where the intended size of the structure is always exactly what's in the name, I'm not sure I'd bother with the variables at all, though others might feel differently.

Reply:
What I meant by "club together" is that since all these structures are either 6, 10, 12, 16 or 32 bytes, we can have variables for these sizes which all these structures can re-use instead of having a separate variable for each structure.

For the second part of your review, I put the constants on top just to be consistent as all other structures have a similar variable. But I agree that its very intuitive to infer the size of each structure due to its name. If others feel the same, then we can go ahead and remove them.

Reply:
I think I agree with Ben's point here. Given that every single static assert is doing the same thing & the size of the struct is in the name, I feel that there's enough indication of the purpose of the "magic" number that it would be fine not to have a variable.

Reply:
Is the removal of constants specific to the structures in this PR or for all other structures as well? (https://github.com/googleinterns/scsi2nvme/pull/20#discussion_r430522862)

Reply:
I also removed it from the other structures because I didn't see the value in extracting a constant since (I predict) we'll only be using that constant in the assert. Another benefit of _not_ having the constant is that you can see the number directly in the assert.

Reply:
I agree that the constants seem a bit superfluous in general. Given the existence of the static asserts, any downstream code that wants to rely on the size of the structs can just invoke `sizeof`, which is more readable anyway.